### PR TITLE
More json cleanup

### PR DESCRIPTION
**Pull request by a544jh**
#75

### CODE REVIEW

File: testResources/json/course.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "api_version": 7,
+    "course": {
+        "id": 3,
+        "name": "test-course",
+        "details_url": "https://example.com/staging/courses/3.json",
+        "unlock_url": "https://example.com/staging/courses/3/unlock.json",
+        "reviews_url": "https://example.com/staging/courses/3/reviews.json",
```
Comment (jamo) [inline testResources/json/course.json]: While at it, we could drop the `staging` word from the urls, maybe just search and replace for all the json files.

File: testResources/json/course.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        "comet_url": "http://example.com:8443/comet",
+        "spyware_urls": ["https://example.com/staging"],
+        "unlockables": [],
+        "exercises": [
+            {
+                "id": 284,
```
Comment (jamo) [inline testResources/json/course.json]: Nit: the #id doesn't really matter, but i guess we could start the numbering from 1...

**Comment by jamo** [comment]
Nice work, added a few nitpicks, feel free to skip those for now if you feel like it.
LGTM - let me know if you feel like doing the few modifications I proposed of if I should merge this as it is.

**Comment by ljleppan** [comment]
I just noticed we are storing some test resources in `/testResources` and some in `/src/test/resources`. Is there any reason to use `/testResources` as `/src/test/resources` is the standard?

**Comment by jamo** [comment]
I thought we even had issue open for moving all test resources to the standard locatio (`/src/test/resources`) but apparently it was for a different repo.

Yes, we should move those too. Either in this one or in a separate PR

**Comment by ljleppan** [comment]
While making changes to the test resources, please also remove [`/src/test/resources/SubmitResponse.json~`](https://github.com/testmycode/tmc-core/blob/master/src/test/resources/submitResponse.json~), or if it's needed for something, remove `~` from the name.

Could probably add a `*~` line to `.gitignore` while we're at it.

File: testResources/json/course.json (hunk 1 above)
Comment (a544jh) [inline testResources/json/course.json]: I think that caused problems with the tests so I had to add it.

**Comment by a544jh** [comment]
The test resources are now moved to the proper location

**Comment by jamo** [comment]
Awesome, merging.

**Pull request merged**